site/content/en/docs/Getting Started/_index.md

```diff
@@ -39,15 +39,34 @@ Here is a sequence diagram of Open Match matching two players.
 
 ## Install the Demo
 
-The demo is installed in the `om-demo` namespace. Run this command to install the Demo:
+This demo deploys the following kubernetes resources.
+
+Under `open-match` namespace:
```

Comment:
The pre-install step already covered the evaluator install. Just replace this bit with:

This step deploys a Match Function and the core demo application in under the `open-match-demo` namespace:

```bash
kubectl create namespace open-match-demo
kubectl apply --namespace open-match-demo \
  -f https://open-match.dev/install/v{{< param release_version >}}/yaml/02-open-match-demo.yaml
```

Reply:
done